draft-ietf-quic-manageability.md

```diff
@@ -991,6 +991,10 @@ While QUIC endpoints are often able to survive address changes, e.g. by NAT
 rebindings, blocking a portion of the traffic based on 5-tuple hashing increases
 the risk of black-holing an active connection when the address changes.
 
+Note that some source ports are assumed to be reflection attack vectors by some
+servers; see {{Section 8.1 of I-D.ietf-quic-applicability}}. As a result, NAT
+rebinding to these source ports can result in that traffic being blocked.
```

Comment:
Is this only about rebinding or any port selection of NATs?

Reply:
I th ink it's mostly about rebinding.

Reply:
I think that it is just "binding" rather than "rebinding".  I read "rebinding" as what happens when flow state is lost and a new port binding is assigned to that flow.  But this is just "binding", which is any time a port is bound to a flow.

Reply:
A quick look around shows a lot more consistent use of 'rebinding' for this case; e.g., [here](https://tools.ietf.org/id/draft-duke-quic-natsupp-00.html)

Reply:
```suggestion
binding to these source ports can result in that traffic being blocked.
```